api/net/Areas/Editor/Models/Storage/FolderModel.cs

```diff
@@ -40,12 +40,10 @@ public FolderModel(string rootPath, string path)
         var files = System.IO.Directory.GetFileSystemEntries(safePath);
         Array.Sort(files, string.CompareOrdinal);
 
-        foreach (var fullName in files)
+        // Don't include temporary folder.
+        foreach (var fullName in files.Where(n => new DirectoryInfo(n).Name != "_tmp"))
         {
-            if (fullName.FileExists() || fullName.DirectoryExists())
```

Comment:
I don't know why we need to validate the file/folder exists?  Line 40 just fetched them, which means they exist.